pages/orders/[orderId]/labels.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Button,
+  Checkbox,
+  CheckboxLabel,
+  Form,
+  FormGroup,
+  Grid,
+  GridItem,
+  Input,
+  Select,
+  Text,
+} from '@bigcommerce/big-design';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const StyledGrid = styled(Grid)`
+  grid-template: 'form print' auto / 1fr auto;
+
+  > .bd-button {
+    position: sticky;
+    top: 0;
+  }
+`;
+
+const OrderLabelPage: React.FC = () => {
```

Comment:
note: the `FC` here is unnecessary. Facebook removed this typecasting a while ago from their both their documentation and official typescript example. Also, FWIW, we're also not doing this on any other page. Please see https://github.com/facebook/create-react-app/pull/8177

Reply:
habit 😅